ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -231,12 +231,13 @@ void shutdown() throws InterruptedException {
 
   /**
    * @param id the {@link PartitionId} to find the store for.
+   * @param skipStateCheck whether to skip checking state of store.
```

Comment:
Introduce `skipStateCheck` makes the interface complex and hard to understand. Can we alway return `store` and check `isStarted()` in the caller?